### DISCUSSION TITLE
Export table to CSV

### DISCUSSION THREAD

**Discussion by JohnnyQQQQ** [grafana/grafana#67988] [Explore]
If today I want to export data from an explore query that is shown in a table, I will first have to create a panel and export it from there. It would be nice if it would directly work from the explore view.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82686. Feel free to continue the discussion around this there. Thank you!
